### ISSUE TITLE
Key/Secret auth fails after upgrade from v1.5.7 to v1.6.6-rc3 with AD auth enabled

### ISSUE THREAD

**Issue by moelsayed** [rancher/rancher#9472]
**Rancher versions:** v1.6.6-rc3
rancher/server: v1.6.6-rc3

**Infrastructure Stack versions:**
healthcheck: v0.2.3
ipsec: v0.11.2
network-services: metadata:v0.9.1 network-manager:v0.6.6
scheduler: - 
kubernetes (if applicable): k8s:v1.5.4-rancher1-4

**Docker version: (`docker version`,`docker info` preferred)**
1.12.6

**Operating system and kernel: (`cat /etc/os-release`, `uname -r` preferred)**
RHEL 7.3
**Type/provider of hosts: (VirtualBox/Bare-metal/AWS/GCE/DO)**
AWS
**Setup details: (single node rancher vs. HA rancher, internal DB vs. external DB)**
single node, internal DB
**Environment Template: (Cattle/Kubernetes/Swarm/Mesos)**
k8s
**Steps to Reproduce:**
* deploy server:v1.5.7
* create k8s environment
* enable AD auth
* create Key/secret and test using rancher cli. Should work with no problems.
* upgrade to server:v1.6.6-rc3

**Results:**
* Key/Secret auth fails after the upgrade:
```
./rancher config
URL [http://1.2.3.4:8080/v2-beta/schemas]: http://111.222.333.444:8080/
Access Key [F8F48999E71FD92D4AC0]: 97BB179995EE0484229
Secret Key [R5H4ATaxuZHRCsHRkdLfiYhoVsfFnQgyVvaoiR4x]: n5RAZRiDB3xMywHP4u7ysgVt3SQezKL3LinwdVLD
FATA[0027] Bad response statusCode [400]. Status [400 Bad Request]. Body: [message=Token malformed after retrieval., baseType=error, code=InvalidFormat] from [http://111.222.333.444:8080/v2-beta]
```
* Creating new keys also doesn't work.

**Comment by deniseschannon** [comment]
Available in v1.6.8-rc1

**Comment by moelsayed** [comment]
Tested upgrade from v1.6.7 to v1.6.8-rc1. Tests passed
